test/logging/README.md

```diff
@@ -0,0 +1,80 @@
+# Prerequisite
+
+- Kubernetes cluster is up and running
+- Install glusterfs-server in all nodes
+
+
+# Create glusterfs cluster
+
+
+- Create glusterfs cluster following instruction on [gluster-ansible](https://github.com/gluster/gluster-ansible) role
+- Create glusterfs endpoing and glusterfs service, refer [this](https://github.com/kubernetes/examples/tree/master/volumes/glusterfs) example
+
+
+# Create PV and PVC
+
+## Create PV
+```
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: glusterfs-pv
+  labels:
+    type: glusterfs
+spec:
+  capacity:
+    storage: 10Gi
+  accessModes:
+    - ReadWriteMany
+  glusterfs:
+    endpoints: glusterfs-cluster
+    path: testvol
+```
+- `path`  should be name of the glusterfs volume
+## Create PVC
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: glusterfs-pvc
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 10Gi
+  volumeName: glusterfs-pv
+```
+
+
+# Create Pod and use PVC
+
+- Build docker image
+```
+docker build -t s3server .
+```
+- Create Pod which continuously logs into configured directory
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: s3server
+spec:
+  containers:
+  - name: s3server
+    image: s3server
+    volumeMounts:
+    - mountPath: "/share/var/log/cortx"
```

Comment:
> "/share/var/log/cortx"

this path needs to be dynamically added btw, after reading from ConfStor, as it may change as per Solution config inputs

Reply:
Added comment